builtin/game/falling.lua

```diff
@@ -56,78 +48,91 @@ core.register_entity(":__builtin:falling_node", {
 		if not vector.equals(acceleration, {x = 0, y = -10, z = 0}) then
 			self.object:set_acceleration({x = 0, y = -10, z = 0})
 		end
-		-- Turn to actual node when colliding with ground, or continue to move
+
 		local pos = self.object:get_pos()
 		-- Position of bottom center point
-		local bcp = {x = pos.x, y = pos.y - 0.7, z = pos.z}
-		-- 'bcn' is nil for unloaded nodes
-		local bcn = core.get_node_or_nil(bcp)
+		local below_pos = {x = pos.x, y = pos.y - 0.7, z = pos.z}
+		-- Avoid bugs caused by an unloaded node below
+		local below_node = core.get_node(below_pos)
```

Comment:
Please undo this change. `get_node_or_nil` is intended.

Reply:
Doing it this way deletes falling entity when inside unloaded area, using the get_node_or_nil ends up leaving falling entity hanging with "Map::getNodeMetadata()" errors.

Reply:
In this case, please ensure that `get_meta` in the node placement code isn't called when the node is still in ignore. I'm against this item loss when the falling node hits ignore, since large machinery builds might rely on this and neither players nor dropped items have this behaviour.

Reply:
It's better that a falling entity stops then starts to fall again when the area is loaded. Falling entities often hit unloaded areas, for example when one falls from a high point on a mountain or floatland.
There must be a better way to approach this.

Reply:
Previous behaviour was to delete the object in true "ignore" at the world's edge, as such objects are not retrievable so are already effectively lost. This PR deletes objects in unloaded/ungenerated areas (which are not "ignore" once loaded/generated), -1.